src/scss/root/_index.scss

```diff
@@ -0,0 +1,102 @@
+@use 'sass:list';
+@use 'sass:map';
+
+@use 'reset';
+@use 'a11y';
+
+$alias: () !default;
+$breakpoints: () !default;
+$global: () !default;
+$tokens: () !default;
+
+@mixin block-components($blockName) {
+	@if list.length(map.get($tokens, 'default', 'blocks', $blockName, 'components')) >= 0 {
+		@each $name, $value in map.get($tokens, 'default', 'blocks', $blockName, 'components') {
+			@if type-of($value) == map {
+				@each $a, $b in $value {
+					--c-#{$name}-#{$a}: #{$b};
+				}
+			}
+		}
+	}
+}
+
+@mixin block-properties($blockName) {
+	@each $name, $value in map.get($tokens, 'default', 'blocks', $blockName) {
+		@if map.has-key($tokens, 'default', 'blocks', $blockName) {
+			@if type-of($value) != map {
+				#{$name}: block-var($blockName, $name);
+			}
+		}
+	}
+}
+
+@mixin component-properties($component) {
+	@if map.has-key($tokens, 'default', 'components', $component) {
+		@each $name, $value in map.get($tokens, 'default', 'components', $component) {
+			#{$name}: component-var($component, $name);
+		}
+	}
+}
+
+@function component-var($component, $item) {
+	@return var(--c-#{$component}-#{$item});
+}
+
+@function block-var($block, $item) {
+	@return var(--b-#{$block}-#{$item});
+}
+
+/* stylelint-disable max-nesting-depth, no-invalid-position-at-import-rule */
```

Comment:
Can we switch all comments to use `//` to avoid additional overhead for Sass compilation and outputting of comments

Reply:
as long as that's not going to get in the way of legal issues... should we include the:
```
/* http://meyerweb.com/eric/tools/css/reset/
   v2.0 | 20110126
   License: none (public domain)
*/
```

Reply:
I suppose since it's public domain, it's free to edit..